numpyro/handlers.py

```diff
@@ -604,16 +604,16 @@ class scope(Messenger):
 
     .. doctest::
 
-        >>> import numpyro
-        >>> import numpyro.distributions as dist
-        >>> from numpyro.handlers import scope, seed, trace
-        >>>
-        >>> def model():
-        ...     with scope(prefix="a"):
-        ...         with scope(prefix="b", divider="."):
-        ...             return numpyro.sample("x", dist.Bernoulli(0.5))
-        ...
-        >>> assert "a/b.x" in trace(seed(model, 0)).get_trace()
+       >>> import numpyro
+       >>> import numpyro.distributions as dist
+       >>> from numpyro.handlers import scope, seed, trace
+       >>>
+       >>> def model():
+       ...     with scope(prefix="a"):
+       ...         with scope(prefix="b", divider="."):
+       ...             return numpyro.sample("x", dist.Bernoulli(0.5))
+       ...
+       >>> assert "a/b.x" in trace(seed(model, 0)).get_trace()
```

Comment:
Using 3 spaces following [doctest docs](https://www.sphinx-doc.org/en/master/usage/extensions/doctest.html#confval-doctest_test_doctest_blocks) and to be consistent w.r.t. to other docstrings in this file. 3 or 4 spaces do not matter I believe.